helper/openssl_certificate_loader.go

```diff
@@ -21,26 +21,65 @@ import (
 	"os"
 
 	"github.com/paketo-buildpacks/libpak/bard"
+	"github.com/paketo-buildpacks/libpak/sherpa"
+	"golang.org/x/sys/unix"
 
 	"github.com/paketo-buildpacks/libjvm"
 )
 
+const TmpTrustStore = "/tmp/truststore"
+
 type OpenSSLCertificateLoader struct {
 	CertificateLoader libjvm.CertificateLoader
 	Logger            bard.Logger
 }
 
+func (o OpenSSLCertificateLoader) prepareTempTrustStore(trustStore, tempTrustStore string) (map[string]string, error) {
+	o.Logger.Infof("Using readonly truststore: %s", tempTrustStore)
+
+	trustStoreFile, err := os.Open(trustStore)
+	if err != nil {
+		return nil, fmt.Errorf("unable to open trust store %s\n%w", trustStore, err)
+	}
+	defer trustStoreFile.Close()
+
+	err = sherpa.CopyFile(trustStoreFile, tempTrustStore)
+	if err != nil {
+		return nil, fmt.Errorf("failed to copy dir (%s, %s)\n%w", trustStore, tempTrustStore, err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to copy dir (%s, %s)\n%w", trustStore, tempTrustStore, err)
```

This is minor, someone before me started the pattern of error messages starting with "unable ...", we try to keep that going.